src/app/containers/CpsFeaturesAnalysis/index.jsx

```diff
@@ -59,11 +80,21 @@ const PromoComponent = ({ promo, dir }) => {
 
   const { isAmp } = useContext(RequestContext);
 
-  const StoryPromo = isAmp || isLive() ? _StoryPromo : FrostedGlassPromo;
+  const promoVariation = useOptimizelyVariation(
+    'high_impact_feature_analysis_promo',
+  );
+
+  let StoryPromoHybrid = StoryPromo;
```

Comment:
This might be more readable

```jsx
const HIGH_IMPACT_VARIATION = 'variation_1' // define this constant outside of the component

// then do
const StoryPromoComponent = HIGH_IMPACT_VARIATION ? FrostedGlassPromo : StoryPromo
```

Reply:
Yeah agreed, changed :)